core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -407,6 +408,10 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       // Delete resource in sam outside of DB transaction
       _ <- workspaceContext.workspace.workflowCollectionName.map( cn => samDAO.deleteResource(SamResourceTypeNames.workflowCollection, cn, userInfo) ).getOrElse(Future.successful(()))
       _ <- samDAO.deleteResource(SamResourceTypeNames.workspace, workspaceContext.workspaceId.toString, userInfo)
+      // Delete workspace manager record (which will only exist if there had ever been a TDR snapshot in the WS)
+      _ = Try(workspaceManagerDAO.deleteWorkspace(workspaceContext.workspaceId, OAuth2BearerToken(gcsDAO.getBucketServiceAccountCredential.getAccessToken), userInfo.accessToken)).recoverWith {
+        case _ => Success(()) //this will only ever succeed if a TDR snapshot had been created in the WS, so we gracefully handle all exceptions here
```

Comment:
could you log, as a warning, any "real" failures here? Any failure where the stub existed, but we failed to delete it.